components/button/style/index.ts

```diff
@@ -90,6 +90,21 @@ export interface ComponentToken {
    * @descEN Horizontal padding of small button
    */
   paddingInlineSM: CSSProperties['paddingInline'];
+  /**
+   * @desc 按钮横向内间距
+   * @descEN Horizontal padding of button
+   */
+  paddingBlock: CSSProperties['paddingInline'];
+  /**
+   * @desc 大号按钮横向内间距
+   * @descEN Horizontal padding of large button
+   */
+  paddingBlockLG: CSSProperties['paddingInline'];
+  /**
+   * @desc 小号按钮横向内间距
+   * @descEN Horizontal padding of small button
+   */
+  paddingBlockSM: CSSProperties['paddingInline'];
```

Comment:
@MadCcc @crazyair Seams like we have wrong English descriptions for these tokens.  All `Horizontal` should be replaced by `Vertical`. I have tried to use these tokens and they change vertical padding of `block` button.